ts/sdk/src/competitionClient.ts

```diff
@@ -413,9 +417,43 @@ export class CompetitionsClient {
 
 	public getCompetitionPublicKey(name: string): PublicKey {
 		const encodedName = encodeName(name);
-		return getCompetitionAddressSync(
-			this.program.programId,
-			encodedName
+		return getCompetitionAddressSync(this.program.programId, encodedName);
+	}
+
+	public async getCompetitionDetails(
+		competition: PublicKey,
+		insuranceFundVaultBalance: BN
```

Comment:
is it possible if we put the logic of getting the `insuranceFundVaultBalance` in this function too, instead of passing it in?

```
		const insuranceVaultBalance = new BN(
			(
				await driftClient.provider.connection.getTokenAccountBalance(
					driftClient.getQuoteSpotMarketAccount().insuranceFund.vault
				)
			).value.amount
		);
```

Reply:
yeah, i just wasnt sure it was worth it or not